docs/source/contributing/docs.rst

```diff
@@ -46,4 +45,35 @@ change renders correctly, it is good practice to test it locally.
    .. tip::
 
       On macOS, you can open a file from the terminal with ``open <path-to-file>``.
-      On Linux, you can do the same with ``xdg-open <path-to-file>``.
\ No newline at end of file
+      On Linux, you can do the same with ``xdg-open <path-to-file>``.
+
+
+.. _contributing/docs/conventions:
+
+Documentation conventions
+=========================
+
+This section lists various conventions we use in our documentation. This is a
+living document that grows over time, so feel free to add to it / change it!
+
+Our entire documentation does not yet fully conform to these conventions yet,
+so help in making it so would be appreciated!
+
+``pip`` invocations
+-------------------
+
+There are three ways to invoke a ``pip`` command:
+
+1. ``pip``
+    This points to python3 pip if you are in an appropriate environment,
+    but in some cases it can actually point to a python2 pip.
+
+2. ``pip3``
+   When present, this always points to a python3 pip, most likely in
+   the same environment as your python3.
+
+3. ``python3 -m pip``
```

Comment:
This should really be the first item here. I would remove 1 and 2 and refer to the pip documentation.

Reply:
that's a good point, +1

Reply:
Maybe even remove the other options and write "There are many ways to run `pip`, this is our preferred option: `python -m pip`"?